frontend/src/component/changeRequest/ChangeRequests/ChangeRequestFilters.tsx

```diff
@@ -1,22 +1,48 @@
-import { Box, Chip, styled } from '@mui/material';
+import { Box, Chip, styled, type ChipProps } from '@mui/material';
 import { useState, type FC } from 'react';
 
-const StyledChip = styled(Chip)(({ theme }) => ({
-    borderRadius: `${theme.shape.borderRadius}px`,
-    padding: theme.spacing(0.5),
-    fontSize: theme.typography.body2.fontSize,
-    height: 'auto',
-    '&[aria-current="true"]': {
-        backgroundColor: theme.palette.secondary.light,
-        fontWeight: 'bold',
-        borderColor: theme.palette.primary.main,
-        color: theme.palette.primary.main,
-    },
-    ':focus-visible': {
-        outline: `1px solid ${theme.palette.primary.main}`,
-        borderColor: theme.palette.primary.main,
-    },
-}));
+const makeStyledChip = (ariaControlTarget: string) =>
+    styled(({ ...props }: ChipProps) => (
+        <Chip variant='outlined' aria-controls={ariaControlTarget} {...props} />
```

Comment:
extract shared props instead of repeating yourself over and over. Will be more useful when we add the open/closed filter too.